### PR TITLE
Update chat mobile demos on android build tools version and READMEs 

### PR DESCRIPTION
**Pull request by ziyiz-amzn**
**Description of changes:**
* Update Android Build Tools version to be compatible with Android Studio 4.0 and later as stated in blog post.
* Update README

**Testing**

1. How did you test these changes? 
Running apps against iOS and android devices. 

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it? Yes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/chat-android/README.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
+This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please read blog posts below to start with various features.
+* [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging)
+* [Push Notifications for Android and iOS with Amazon Chime SDK messaging](https://aws-blogs-prod.amazon.com/business-productivity/push-notifications-for-android-and-ios-with-amazon-chime-sdk-messaging/)
```
Comment (hokyungh) [inline apps/chat-android/README.md:7]: This link looks weird.

File: apps/chat-android/build.gradle [hunk 2]
```diff
@@ -16,7 +16,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```
Comment (hokyungh) [inline apps/chat-android/build.gradle:19]: why downgraded?

File: apps/chat-ios/Podfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '13.0'
+# platform :ios, '9.0'
```
Comment (hokyungh) [inline apps/chat-ios/Podfile:2]: so what's minimum version we should support? Same as Chime which is iOS 10?

File: apps/chat-ios/README.md [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
+This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please read blog posts below to start with various features.
+* [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging)
+* [Push Notifications for Android and iOS with Amazon Chime SDK messaging](https://aws-blogs-prod.amazon.com/business-productivity/push-notifications-for-android-and-ios-with-amazon-chime-sdk-messaging/)
```
Comment (hokyungh) [inline apps/chat-ios/README.md:7]: link looks wrong

File: apps/chat-ios/README.md [hunk 5]
```diff
@@ -2,21 +2,21 @@
...
-2. You have completed the following operations
-    1. [Deploy the solution](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
-    2. [Run the Amazon Chime SDK Chat Demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#running-the-amazon-chime-sdk-chat-demo).
-    3. [Create Amazon Cognito Users](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#cognito-user-pools) OR [Use Credential Exchange Service](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#credential-exchange-service) to join a chat session.
-3. You have installed Xcode version 11.0 or later.
-4. You have [CocoaPods](https://cocoapods.org/) installed.
+1. [Deploy the AWS CloudFormation Template](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
+2. [Run the Amazon Chime SDK Chat Web Demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#running-the-amazon-chime-sdk-chat-demo).
```
Comment (hokyungh) [inline apps/chat-ios/README.md:16]: nit: title and tag seems mistach `running-the-amazon-chime-sdk-chat-demo` vs `Run the Amazon Chime SDK Chat Web Demo`

File: apps/chat-ios/README.md [hunk 6]
```diff
@@ -2,21 +2,21 @@
...
 
-1. You have read [Build chat features into your application with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-features-into-your-application-with-amazon-chime-sdk-messaging/). You understand the architecture of the chat demo web application built with Amazon Chime SDK messaging. 
-2. You have completed the following operations
-    1. [Deploy the solution](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
-    2. [Run the Amazon Chime SDK Chat Demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#running-the-amazon-chime-sdk-chat-demo).
-    3. [Create Amazon Cognito Users](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#cognito-user-pools) OR [Use Credential Exchange Service](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#credential-exchange-service) to join a chat session.
-3. You have installed Xcode version 11.0 or later.
-4. You have [CocoaPods](https://cocoapods.org/) installed.
+1. [Deploy the AWS CloudFormation Template](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
```
Comment (hokyungh) [inline apps/chat-ios/README.md:15]: nit: title and tag seems mismatch `deploying-the-solution` vs `Deploy the AWS CloudFormation Template`

File: apps/chat-android/README.md (hunk 1 above)
Comment (ziyiz-amzn) [reply apps/chat-android/README.md:7]: yeah, will update

File: apps/chat-android/build.gradle (hunk 2 above)
Comment (ziyiz-amzn) [reply apps/chat-android/build.gradle:19]: AS 4.0 does not support gradle 4.2.2, but Android Studio 4.0 and later is stated as requirement in the blog post.

File: apps/chat-ios/Podfile (hunk 3 above)
Comment (ziyiz-amzn) [reply apps/chat-ios/Podfile:2]: CocoaPods provides a default deployment target if one is not specified. The current default values are 4.3 for iOS

Comment (ziyiz-amzn) [reply apps/chat-ios/Podfile:2]: It was commented before and we do not specify any iOS version. Changed to 13 while testing local build but it did not make any difference.

File: apps/chat-ios/README.md (hunk 5 above)
Comment (ziyiz-amzn) [reply apps/chat-ios/README.md:16]: It's fine I think. Better description here in readme

**Review by hokyungh — APPROVED** [review]

**Pull request merged**